project-generator/src/main/java/com/redhat/ipaas/project/converter/DefaultProjectGenerator.java

```diff
@@ -162,6 +172,18 @@ public DefaultProjectGenerator(ConnectorCatalog connectorCatalog, ProjectGenerat
                     continue;
                 }
 
+                if ( MAPPER.equals(step.getStepKind()) ) {
```

Comment:
Seems that one of my last comments got lost (probably didn't submitted it and cant find it anymore): Actually instead of having here many conditionals I'd prefer some sort of 'handle' abstraction which is looked up via the step kind. This would make the code smaller and easier to understand. also you don't have this nasty loop control via `continue`